src/server.js

```diff
@@ -91,6 +92,20 @@ function init(socket, client, token) {
 		socket.emit("auth");
 		socket.on("auth", auth);
 	} else {
+		var ip = socket.handshake.headers['x-forwarded-for'] || socket.request.connection.remoteAddress;
```

Comment:
If you are exposing node.js to the world on port 80 without a proxy you are pretty much inviting what happens.